agent/agent.go

```diff
@@ -967,6 +970,7 @@ func (a *agent) handleReconnectingPTY(ctx context.Context, logger slog.Logger, m
 
 	connectionID := uuid.NewString()
 	logger = logger.With(slog.F("id", msg.ID), slog.F("connection_id", connectionID))
+	logger.Debug(ctx, "handling ReconnectingPTY")
```

Comment:
```suggestion
	logger.Debug(ctx, "starting handler")
```
(reconnecting-pty already covered in logger name)